exorelay-go/bin/lint

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+golint
```

Comment:
Cannot mimic how morula does `lint` since this directory is not in `$GOPATH/srs/github.com/Originate/exorelay-go` 

Unless, what if we change the import path to `github.com/Originate/exocom/exorelay-go`? Even though exocom is not a Go codebase.